tests/next_tests/integration_tests/feature_tests/iterator_tests/test_builtins.py

```diff
@@ -331,6 +333,11 @@ def test_can_deref(program_processor, stencil):
 @pytest.mark.parametrize("as_column", [False, True])
 def test_cast(program_processor, as_column, input_value, dtype, expected_value):
     program_processor, validate = program_processor
+    if program_processor == run_dace_iterator:
+        # bug in DaCe translation to C-code with implicit cast on nested SDFG boundary
+        # this bug disappears after inline_sdfgs pass (part of simplify)
+        pytest.xfail("Not supported in DaCe backend: implicit cast")
```

Comment:
I propose to apply `inline_sdfgs` to the final SDFG, in order to enable several test cases. The `inline_sdfgs` pass is simple enough to be considered safe, and it should make compilation easier by reducing the level of nesting in function calls.